.github/STYLEGUIDE.md

```diff
@@ -743,6 +743,37 @@ Each guide is broken into a few sections. Sections contain a list of guidelines.
   ```
   </details>
 
+* **PREFER attributes on the same line as what they are attributing.**
```

Comment:
Is this the rule that should also dictate order of attributes?  Is that a rule we need?
```swift
@MainActor @ViewBuilder func createViews() {
```
or
```swift
@ViewBuilder @MainActor func createViews() {
```

Reply:
The order of attributes should be the same kind of rules that we have for other ordering, based off of effect on the overall codebase. So `@MainActor` affects every call site, `@discardableResult` does as well. `@ViewBuilder` only affects that function really, and `@inlineable` is a somewhat transparent compiler optimization.

Reply:
I think this makes sense.  +1 to Same rules as our file organization

Reply:
`@MainActor @ViewBuilder func createViews() {` was added as an example. Should the ordering preference be a part of the existing preferences or are we proposing a 3rd addition to call out order of attributes?

Reply:
I think a 3rd rule is needed for ordering.  It essentially repeats or summarizes the rule for file organization.  The rule holds true despite being in the stacked or inline format.  Items with the most impact come first.

Reply:
This might be covered by the "Why" of "DO place content in the correct order..." but if we need more clarity I'm on board with either expanding the existing or creating a new "DO" for ordering attributes.

Reply:
Expanding the existing one would be enough.  Just something to point at that says "Do it this way"

Reply:
Expanded the examples of the correct ordering guideline.